Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,204 @@
+            _root = rootVisualElement;
+            _mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_mVisualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _mVisualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            ApplyText();
+
+            CoreApi = CoreApi.SharedInstance;
+            var awsCredentials = AwsCredentialsFactory.Create();
+            CreationModel = awsCredentials.Creation;
+            UpdateModel = awsCredentials.Update;
+            RefreshProfiles();
+            
+            var contentContainer = _root.Q(className: "main__content").Children().First();
+            var landingPage = new LandingPage(contentContainer);
+            _userProfilesPage = new AwsUserProfilesPage(contentContainer,this);
+
+            _tabButtons = _root.Query<Button>(className: TabButtonClassName).ToList();
+            _tabContent = _root.Query(className: TabContentClassName).ToList();
+
+            _tabButtons.ForEach(button => button.RegisterCallback<ClickEvent>(_ => { OpenTab(button.name); }));
+
+            OpenTab("Landing");
+        }
+
+        private void ApplyText()
+        {
+            var l = new ElementLocalizer(_root);
+            l.SetElementText("Landing", Strings.TabLanding);
+            l.SetElementText("Credentials", Strings.TabCredentials);
+            l.SetElementText("Anywhere", Strings.TabAnywhere);
+            l.SetElementText("EC2", Strings.TabEC2);
+            l.SetElementText("Help", Strings.TabHelp);
+        }
+
+        private void OpenTab(string tabName)
+        {
+            _tabContent.ForEach(page =>
+            {
+                if (!page.name.StartsWith(tabName))
+                {
+                    page.RemoveFromClassList(TabContentSelectedClassName);
+                }
+                else
+                {
+                    page.AddToClassList(TabContentSelectedClassName);
+                }
+            });
+
+            _tabButtons.ForEach(button =>
+            {
+                if (button.name != tabName)
+                {
+                    button.RemoveFromClassList(TabButtonSelectedClassName);
+                }
+                else
+                {
+                    button.AddToClassList(TabButtonSelectedClassName);
+                }
+            });
+        }
+        
+        public void RefreshProfiles()
+        {
+            UpdateModel.Refresh();
+            CurrentState.AllProfiles = UpdateModel.AllProlfileNames;
+        }
+        
+        public void SetupWrapper()
+        {
+            var credentials = CoreApi.RetrieveAwsCredentials(CurrentState.SelectedProfile);
+            var client = new AmazonGameLiftClient(credentials.AccessKey, credentials.SecretKey);
+            GameLiftWrapper = new AmazonGameLiftWrapper(client);
+        }
+        
+        public void OpenS3Popup(string bucketName)
```

Comment:
Same for this. I don't see this referenced anywhere?

Reply:
This is referenced within AwsCredentialsProfilePage.cs